plugin/test/border.js

```diff
@@ -0,0 +1,200 @@
+import { setup } from "./_helpers.js";
+import { describe, expect, test } from "vitest";
+
+setup();
+
+describe("border", () => {
+  test("supports x|y with value", async (t) => {
+    const x = ["border-x-0", "border-x-2", "border-x-4", "border-x-8"];
+    const y = ["border-y-0", "border-y-2", "border-y-4", "border-y-8"];
+
+    const classes = [...x, ...y];
+
+    const { css } = await t.uno.generate(classes);
+
+    expect(css).toMatchInlineSnapshot(`
+      "/* layer: default */
+      .border-x-0{border-left-width:0px;border-right-width:0px;border-left-style:solid;border-right-style:solid;}
+      .border-x-2{border-left-width:2px;border-right-width:2px;border-left-style:solid;border-right-style:solid;}
+      .border-x-4{border-left-width:4px;border-right-width:4px;border-left-style:solid;border-right-style:solid;}
+      .border-x-8{border-left-width:8px;border-right-width:8px;border-left-style:solid;border-right-style:solid;}
+      .border-y-0{border-top-width:0px;border-bottom-width:0px;border-top-style:solid;border-bottom-style:solid;}
+      .border-y-2{border-top-width:2px;border-bottom-width:2px;border-top-style:solid;border-bottom-style:solid;}
+      .border-y-4{border-top-width:4px;border-bottom-width:4px;border-top-style:solid;border-bottom-style:solid;}
+      .border-y-8{border-top-width:8px;border-bottom-width:8px;border-top-style:solid;border-bottom-style:solid;}"
+    `);
+  });
+
+  test("supports setting border width", async (t) => {
```

Comment:
Looks like missing a test for just `border` as a class (unless I missed it, it's super hard to search for) 🙈

Reply:
border-b, border-t coming up ⚡